tools/cache.py

```diff
@@ -37,7 +37,7 @@ def __init__(self, dirname):
     self.filelock = filelock.FileLock(self.filelock_name)
 
   def acquire_cache_lock(self):
-    if config.FROZEN_CACHE:
+    if config.FROZEN_CACHE and not settings.WASMFS:
```

Comment:
Hi @sbc100 Is there a good way to include `libwasmfs` and `libwasmfs-mt` when running `embuilder build ALL`? Currently `libwasmfs` is not included in `system_libs` unless the `settings.WASMFS` flag is true. 

Would making a separate `embuilder build LIBWASMFS` and including it in the `ci` tests be a good idea?